examples/bindist/Makefile

```diff
@@ -18,13 +18,14 @@ QUIET ?= 1
 # bindist specific stuff:
 #
 # build and use module "abc".
-# use BINARY_DIRS instead of DIRS
-BINARY_DIRS += abc
-USEMODULE += abc
+# use BIN_DIRS instead of DIRS, BIN_USEMODULE instead of USEMODULE,
+# to flag that the directory should only be included when doing a normal build,
+# and that the resulting .a should be saved when doing "make bindist"
+BIN_DIRS += abc
+BIN_USEMODULE += abc        # include "abc" module
+BIN_USEMODULE += bindist    # include application module (named $APPLICATION)
```

Comment:
why is this line necessary? It seems to also work without this line. Could you elaborate on its function?

Reply:
This selects the module "bindist", which in this case is the application, to be included in the binary distribution.